src/Query/util.ts

```diff
@@ -80,46 +80,14 @@ export const extractFacetFilters = (searchParams: URL['searchParams']): Filter<D
   return filter;
 };
 
-export const getFacetsForMeta = (filter: Filter<Document>, taxonomy: FacetDisplayNames) => {
-  const facets: { [key: string]: { type: 'string'; path: string } } = {};
+// TODO: update this to work with new facet structure (children and options)
+export const getFacetAggregationStages = (taxonomy: FacetOption[]) => {
+  const facetKeysForAgg: FacetAggregationStage = {};
 
-  for (const baseFacet in taxonomy) {
-    facets[baseFacet] = {
-      type: 'string',
-      path: `facets.${baseFacet}`,
-    };
+  // TODO: traverse all of taxonomy to get facets for meta
+  for (const facetOption of taxonomy) {
+    console.log(facetOption);
```

Comment:
Do we want to keep this logged?

Reply:
removed in follow up! https://github.com/mongodb/docs-search-transport/pull/77/files#diff-244d01de0824eda50531865572bc5dfaf1c3325fcb645aa6012177cfbdbb1017L87